action/serial.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * DokuWiki Plugin struct (Action Component)
+ *
+ * @license GPL 2 http://www.gnu.org/licenses/gpl-2.0.html
+ * @author  Andreas Gohr, Michael Große <dokuwiki@cosmocode.de>
+ */
+
+use dokuwiki\plugin\struct\meta\AccessTable;
+use dokuwiki\plugin\struct\meta\AccessTableData;
+use dokuwiki\plugin\struct\meta\Column;
+use dokuwiki\plugin\struct\meta\SerialTable;
+use dokuwiki\plugin\struct\meta\Schema;
+use dokuwiki\plugin\struct\meta\SearchConfig;
+use dokuwiki\plugin\struct\meta\StructException;
+use dokuwiki\plugin\struct\meta\Value;
+
+/**
+ * Class action_plugin_struct_serial
+ *
+ * Handle serial editing
+ */
+class action_plugin_struct_serial extends DokuWiki_Action_Plugin
```

Comment:
This class looks a lot like `action_plugin_struct_lookup`. I guess we could merge them into one or have them both inherit from a common ancestor.